recipes/scheduler.rb

```diff
@@ -15,11 +12,39 @@
 template node['snapraid']['scheduler']['config_file'] do
 end
 
-# Setup our cronjob
-cron 'snapraid-runner' do
-  hour node['snapraid']['scheduler']['execution_hour']
-  minute node['snapraid']['scheduler']['execution_minute']
-  command "sudo python #{node['snapraid']['scheduler']['script_directory']}/snapraid-runner.py "\
-    "-c #{node['snapraid']['scheduler']['config_file']}"
-  user 'root'
+#
+# Install Python & configure scheduled task (Windows)
+#
+if platform?('windows')
+  windows_package 'python2.7' do
```

Comment:
Something to keep in mind for the future. There may be a python cookbook that could be used here. Could potentially cause issues for other cookbooks.